### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by dmcdenissen** [beyondcode/herd-community#608]
### Platform

Windows

### Operating system version

Windows 11 (Version 22H2)

### System architecture

Windows

### Herd Version

1.0.1

### PHP Version

?

### Bug description

I installed Laravel Herd on windows (I have PRO). But when making a new site it says it has made the site but nothing happens. 
I think this is because there is no local PHP installed, when I use powershell and run `php -v` it says the term is not recognized.

**Comment by mechelon** [comment]
Can you try and restart Herd? Might be related to PATH changes not being properly detected.

**Comment by wipeouttom** [comment]
Colleague of @dmcdenissen here.

Reading other issues on we came to the conclusion that PATH values were not set correctly. Our local accounts don't have install permissions so we have to use a different account for that. The PATH changes were applied to the account with install permissions. So we had to apply the changes to our local user account environment values so the PATH values are added.

**Comment by dmcdenissen** [comment]
Indeed, our organisation uses a local admin account other then our useraccount. When installing Herd it seems to set the PATH environment of herd onto the local admin account... After changing it all works well ...
